src/_pytest/config/__init__.py

```diff
@@ -793,9 +831,9 @@ class InvocationParams:
             Plugins accessing ``InvocationParams`` must be aware of that.
         """
 
-        args = attr.ib(converter=tuple)
+        args = attr.ib(type=Tuple[str, ...], converter=_args_converter)
```

Comment:
Explicit converter instead of `tuple` needed just for the type annotation to work